lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -428,6 +428,33 @@ struct ExprVisitor {
     return builder.create<moore::ExtractOp>(loc, type, value, lowBit);
   }
 
+  Value visit(const slang::ast::MemberAccessExpression &expr) {
+    llvm::SmallVector<const slang::ast::ValueSymbol *> symbolVec;
+    const auto *exprIt = &(expr.value());
+    std::string concatName(expr.member.name);
+    symbolVec.push_back(
+        static_cast<const slang::ast::ValueSymbol *>(&expr.member));
+    while (true) {
+      std::string structName((*exprIt).getSymbolReference()->name);
+      concatName = structName + "." + concatName;
+      symbolVec.push_back(static_cast<const slang::ast::ValueSymbol *>(
```

Comment:
I think we could create `SturctFieldAccessOp` here.

Reply:
Thanks for your  reply, I will consider.